run_cam.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+. venv/bin/activate && python3 camdaemon.py
```

Comment:
it doesn't matter too much because we'll redo this in #10, but the proper way to do this when using systemd (and CLI like here) is to specify the full path to the python binary.  That then will pick relative installs libraries inside the virt env.

so this might be:

```suggestion
/root/picamserver/venv/bin/python3 camdaemon.py
```

Reply:
Agree that absolute directories are way less sketchy, but maybe ~/picamserver/venv/etc instead of /root/? It doesn't require root perms, and we don't know the username.

I'm somewhat inclined to punt on this until #10, though. Open to thoughts.

Reply:
ah - yeah - let's be opinionated with `~/picamserver` and then tell yell out in docs "we use `picamserver` as user, salt to taste!"  . And totes - love being secure by default and not running as `root`.  I'm not doing that right now, no sir, not at all.